### PR TITLE
Add new fields to classes

### PR DESCRIPTION
**Pull request by filipeferreirasoriano**
Added fields entryDate and studentGroup to InvitationRequestDTO, Invitation model and Student model.

### CODE REVIEW

File: src/main/java/com/academy/edge/studentmanager/dtos/InvitationRequestDTO.java [hunk 1]
```diff
@@ -4,11 +4,16 @@
...
 
 @Data
 public class InvitationRequestDTO {
 
     @EmailCollection
     private List<String> emails;
+
+    private int studentGroup = 1;
+
+    private String entryDate = LocalDate.now().toString().substring(0, 8) + "01";
```
Comment (daviromao) [inline src/main/java/com/academy/edge/studentmanager/dtos/InvitationRequestDTO.java:18]: Remvoer os valores default do DTO

File: src/main/java/com/academy/edge/studentmanager/repositories/InvitationRepository.java [hunk 2]
```diff
@@ -7,4 +7,6 @@
 @Repository
 public interface InvitationRepository extends CrudRepository<Invitation, String> {
     Invitation findByCodeAndEmail(String invitationId, String email);
+
+    Invitation findByEmail(String email);
```
Comment (daviromao) [inline src/main/java/com/academy/edge/studentmanager/repositories/InvitationRepository.java:11]: Remove.

**Comment by filipeferreirasoriano** [comment]
screenshot_teste_1
screenshot_teste_2
screenshot_teste_3

**Comment by daviromao** [comment]
Valeu @filipeferreirasoriano, ficou ótima as mudanças

**Pull request merged**